app/views/bills/_form.html.erb

```diff
@@ -22,11 +34,13 @@
     <%= f.text_field :usage, :id => "usage" %>
   </div>
 
-  <%= f.fields_for :units do |units_form| %>
-    <div class="field">
-      <%= units_form.label :number_occupants, "Number of Occupants in Your Home or Apartment" %><br>
-      <%= units_form.text_field :number_occupants, :id => "occupants" %>
-    </div>
+  <% if @show_occupants %>
```

Comment:
That's how I did it originally, but in this case I decided to move the tricky conditional into the controller so the view would be more readable and have less processing to do. I just added another commit that refactors the variable assignment in the controller to 1 line, which may help. If you still want me to, I can definitely move this into the view - I'm just going off the general practice of keeping as little logic as possible in the view.